src/components/Sidebar/Sidebar.jsx

```diff
@@ -1,26 +1,27 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import './Sidebar.css';
 
 export const Sidebar = () => {
   return (
     <>
       <aside className='sidebar'>
         <ul className='sidebar__menu'>
-          <li className='sidebar__link text active'>
-            <i className='fas fa-home'></i>Home
-          </li>
-          <li className='sidebar__link text'>
+          <Link to='/video' className='sidebar__link text active'>
```

Comment:
You can use `NavLink` for using activeClass method to toggle the css when the link is active